### ISSUE TITLE
5.1.0 breaks facepile

### ISSUE THREAD

**Issue by bekopharm** [pfefferle/wordpress-webmention#407]
My facepiles are gone when I upgrade von `5.0.0` to `5.1.0`:

![image](https://github.com/pfefferle/wordpress-webmention/assets/5016428/a5126a7c-6768-4240-928f-44ac8d14f8cd)
(This whole section is completely missing after updating)

Using Sempress theme with a `comments.php` if that helps.

WP is `6.2.2`
That are my settings:

![image](https://github.com/pfefferle/wordpress-webmention/assets/5016428/a55763d2-7b2a-46b0-b8fd-aad24e3f09fb)

Nothing suspicious in the error log.

**Comment by pfefferle** [comment]
Have you rolled back? There seem to be no issue on this page for example: https://beko.famkos.net/2023/04/04/running-x4-foundations-with-my-simpit/

**Comment by bekopharm** [comment]
Yes I did roll back. Always making a snapshot before touching WordPress updates :sweat:

**Comment by bekopharm** [comment]
Found it. @janboddez comment (https://github.com/pfefferle/wordpress-webmention/issues/404#issuecomment-1591191623) made it click:

I removed `comment_form()` from my theme in `comments.php` years ago because I _do not want_ to offer a comment field at all. I only want Webmentions and ActivityPub reactions. 

Thus the removal of `add_filter( 'comments_template', array( static::class, 'filter_comments_template' ) );` in `class-comment-walker.php` while introducing the action `comment_form_before` created this issue for me because `comment_form_before` was never called without  `comment_form()`.

I'm doing this now manually in my `comments.php` of the SemPress theme here:

```php
	<?php // You can start editing here -- including this comment! ?>
	
	<?php do_action( 'comment_form_before' ); ?>

	<?php if ( have_comments() ) : ?>
```

So… solved :grin: 
Running now `5.1.1` with facepiles.
